cfgmgr/intfmgr.cpp

```diff
@@ -478,6 +509,21 @@ bool IntfMgr::doIntfGeneralTask(const vector<string>& keys,
             }
         }
 
+        if (!grat_arp.empty())
+        {
+            if (!setIntfGratArp(alias, grat_arp))
+            {
+                SWSS_LOG_ERROR("Failed to set ARP accept to \"%s\" state for the \"%s\" interface", grat_arp.c_str(), alias.c_str());
+                return false;
+            }
+
+            if (!alias.compare(0, strlen(VLAN_PREFIX), VLAN_PREFIX))
+            {
+                FieldValueTuple fvTuple("arp_accept", grat_arp);
```

Comment:
The field must be same as what is in config_db - "grat_arp". Lets not change it.